lua/telescope/actions/init.lua

```diff
@@ -225,6 +225,48 @@ actions.run_builtin = function(prompt_bufnr)
   require('telescope.builtin')[entry.text]()
 end
 
+-- Maybe upstream in plenary. TJ was already working on a new Path module
+-- Or maybe tami5 is interested.
+local create_file = function(file)
+  vim.loop.fs_open(file, "w", 420, vim.schedule_wrap(function(err, fd)
+    if err then
+      vim.api.nvim_err_writeln('Could not create file: ' .. file)
+    else
+      vim.loop.fs_close(fd)
+    end
+  end))
+end
```

Comment:
@tami5 Do i do that correctly. Also comment

Reply:
Yeb that looks correct. Also regarding the comment, I was under the impression that we will create a new filesystem repo for telescope. But if you want us to temporary have action/fs.lua I can work on it

Reply:
I don't want `actions/fs.lua`. We had discussed just opening and creating here so its usable and all that other fancy stuff in a extension. But i was thinking doing that `fs_` stuff in a plenary module and than just using that abstraction for `telescope-fs.nvim`

Reply:
hmm 🤔, okay I’ll create a pr in plenary but will this pr depend on it because I only can work on it tomorrow.

My suggestion is to merge asap and the missing basic fs functions can be added when plenery pr is merged

Reply:
Thats https://github.com/nvim-lua/plenary.nvim/pull/22 the current work of tj. (You are also probably only interested in the first commit. Everything else is github actions)
That was also just an idea. Does not need to be rushed. We can replace that raw `fs_` stuff i am currently doing here later. This PR one is also not done yet. Kinda wanna be able to create nested directories and directories + file. See lines 253 - 260

Reply:
@Conni2461 plenary has a full set of fs related stuff, is there something in particular we want me to implement in plenary path module?

Reply:
Oh. i should have looked before writing. I thought we only had a WIP module. My bad

Reply:
Check it and tell me if something you like on it, so I can create it :smile: